src/runtime_src/core/pcie/driver/linux/xocl/subdev/xgq_vmr.c

```diff
@@ -1200,6 +1202,15 @@ static int xgq_refresh_system_dtb(struct xocl_xgq_vmr *xgq)
 		&xgq->xgq_vmr_system_dtb_size, XGQ_CMD_LOG_SYSTEM_DTB, 0, 0);
 }
 
+static int xgq_refresh_shell_int_uuid(struct xocl_xgq_vmr *xgq)
```

Comment:
when do you call this function to reload interface uuid?

Reply:
We can add a sysfs node to refresh this info.

Reply:
> We can add a sysfs node to refresh this info.

This does not make sense to me. The refresh should only be called when the shell has changed. Driver knows exactly when the shell can change, so driver should not rely on any other triggers to refresh. We can call it in probe, and after hot reset?

Reply:
ok, we will add this check in the probe, it is good for both cold reboot and hot reset.

Reply:
fixed